canonical-websites/www.ubuntu.com
# Draft of the new /embedded page

## Done

- Created a quick demo of the /embedded page

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [/embedded](http://0.0.0.0:8001/embedded)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- Compare to the [copy doc](https://docs.google.com/document/d/15Tz87YRg4SnX4DWGOSl2TDTHLSStJoma879xHnF0nuo/edit#heading=h.c6wjbvvp8mcm)


## Issue / Card

Fixes #4864
